plugins/ModelChecker/ModelChecker.qml

```diff
@@ -14,7 +14,7 @@ UM.SimpleButton
 
     width: UM.Theme.getSize("save_button_specs_icons").width
     height: UM.Theme.getSize("save_button_specs_icons").height
-    iconSource: "model_checker.svg"
+//    iconSource: "model_checker.svg"
```

Comment:
We should still have an icon on this button, right?

Looks like this property still exists and everything, so I'll suggest just uncommenting it. But I haven't tested that and there was probably a reason why this was disabled.
```suggestion
    iconSource: "model_checker.svg"
```